content/engine/security/rootless.md

```diff
@@ -58,6 +58,40 @@ testuser:231072:65536
   ([Ubuntu-specific kernel patch](https://kernel.ubuntu.com/git/ubuntu/ubuntu-bionic.git/commit/fs/overlayfs?id=3b7da90f28fe1ed4b79ef2d994c81efbc58f1144)).
 
 - Known to work on Ubuntu 18.04, 20.04, and 22.04.
+
+- Ubuntu 24.04 and later enables restricted unprivileged user namespaces by
+  default, which prevents unprivileged processes in creating user namespaces
+  unless an AppArmor profile is configured to allow programs to use
+  unprivileged user namespaces. If you install the rootless extras using the
+  [installation script](https://get.docker.com/rootless), you must add an
+  AppArmor profile for `rootlesskit`.
+
+  1. Add the AppArmor profile to `/etc/apparmor.d/usr.local.bin.rootlesskit`:
+
+     ```console
+     $ cat <<EOF > /etc/apparmor.d/$(echo $HOME/bin/rootlesskit | sed -e s@^/@@ -e s@/@.@g)
+     abi <abi/4.0>,
+     include <tunables/global>
+
+     $HOME/bin/rootlesskit flags=(unconfined) {
+       userns,
+
+       include if exists <local/$(echo $HOME/bin/rootlesskit | sed -e s@^/@@ -e s@/@.@g)>
+     }
+     EOF
+     ```
+
+  2. Restart AppArmor.
+
+     ```console
+     $ systemctl restart apparmor.service
+     ```
+
+  This is not necessary you if you install `docker-ce-rootless-extras` using
+  the deb package (`apt-get install docker-ce-rootless-extras`), since the
+  AppArmor profile for `rootlesskit` is already bundled with the `apparmor` deb
+  package.
```

Comment:
Maybe we should move this before the actual instructions? Maybe even add it as a [!NOTE] so it's not easy to miss?
I might imagine someone first following the instructions and then finding out that it wasn't actually needed 😅